integrationtest/org/daisy/dotify/formatter/test/resource-files/marker-ref/marker-reference-across-sequence-workaround-input.obfl

```diff
@@ -9,12 +9,12 @@
          <header/>
          <footer>
             <field>
-               <marker-reference marker="foo" direction="backward" scope="sequence"/>
```

Comment:
Should this file still be called "workaround"?

Reply:
I would still call this a workaround yes, because `marker-reference` doesn't search across sequences (scope "document" exists but is not implemented).